HOW_TO.md

```diff
@@ -91,13 +99,17 @@ If there are other dependencies needed by your algorithm, add them to the `packa
 The packages listed will be installed using `apt-get` as part of the Docker build process.
 
 ### Test your algorithm <a name="test"/>
+
 A testing script named `testing.py` is provided for testing your algorithm.
 It checks whether the configuration is correct for testing the files by making sure that the arguments in algorithm_rgb as well as the image files are in the correct format
 
 The testing script requires `numpy` and `gdal` to be installed on the testing system.
 
 If your files reside in `/user/myself/test_images` the command to test could be the following:
-```./testing.py /user/myself/test_images```
+
+```
+./testing.py /user/myself/test_images
```

Comment:
While it's not needed, if `python3` is added to the other python script commands it should be added here as well

```suggestion
python3 testing.py /user/myself/test_images
```